memory/player_party_manager.py

```diff
@@ -1,14 +1,25 @@
+from enum import Enum
+
 from engine.mathlib import Vec3
 from memory.core import mem_handle
 
 
+# PlayerDefaultState.EState
+class PlayerState(Enum):
```

Comment:
Possibly name this `PlayerMovementState` or something similar, since `PlayerState` doesn't really tell us what it is.